pkg/refproto/ssl_game_controller_auto_ref.proto

```diff
@@ -19,6 +19,29 @@ message AutoRefToController {
     optional Signature signature = 1;
     // game_event is an optional event that the autoRef detected during the game
     optional GameEvent game_event = 2;
+
+    // the origin of this game event
+    // autoRef id
+    optional string origin = 5;
```

Comment:
Why do we need this again when it is already present in the AutoRefRegistration? (Not that it would be hard to add though)

Reply:
good point. I'll remove it again.